hardware_enablement/about-hardware-enablement.adoc

```diff
@@ -0,0 +1,13 @@
+[id="about-hardware-enablement"]
+= About the OpenShift SDN default CNI network provider
+include::modules/common-attributes.adoc[]
+:context: about-openshift-sdn
+
+toc::[]
+
+Many applications require specialized hardware or software which depend on kernel modules or drivers. The recommended approach to load out-of-tree kernel modules on {op-system-first} nodes is to use driver containers. To deploy  out-of-tree drivers at the same time as cluster installation (day-1) the kmods-via-containers framework can be used. For loading drivers or kernel modules on an existing {product-title} cluster (day-2) {product-title} offers several tools:
+
+* The Node Feature Discovery (NFD) Operator for adding node labels for CPU capabilities, kernel version, PCIe device vendor IDs, and more. For information about installing and using the NFD Operator, see xref:./psap-node-feature-discovery-operator[the Node Feature Discovery Operator]
+* The Driver Toolkit is a container image which is a part of every OpenShift release. It contains the kernel packages and other common dependencies needed for building a driver or kernel module. The Driver Toolkit can be used as a base image for driver container image builds on the {product-title}. For more information about using the Driver Toolkit, see xref:./psap-driver-toolkit[the Driver Toolkit].
+* The Special Resource Operator (SRO) orchestrates the building and management of driver containers for loading kernel modules and drivers on an existing (day 2) OpenShift or Kubernetes cluster. For more information about the Special Resource Operator see xref:./psap-special-resource-operator[the Special Resource Operator].
+
```

Comment:
Remove xrefs in a module. (Remove 3 occurences "For more information"....)
Reorder bullets to match order in the book (replaces the need for xrefs):
   Driver Toolkit
   SRO
   NFD